src/helpers/getByAPI.js

```diff
@@ -1,24 +1,44 @@
 // @flow
 import * as React from 'react';
+import prettyFormat from 'pretty-format';
 import ErrorWithStack from './errorWithStack';
 
+const getNodeByType = (node, type) => node.type === type;
```

Comment:
I think it will be better to name `getNodeByName` and `getNodeByType` as `filterNodeByName` and `filterNodeByType`. The `getNode` method indicates that it returns `Node`, but in fact it returns `Boolean`